### PR TITLE
Fixed validation

### PR DESCRIPTION
**Pull request by DanielElisenberg**
# FIXED VALIDATION

When I started generating/restructuring the exampledata I used the data @vaskalan and me made for the test_datastore.
the SYNT_BEFOLKNING_KJOENN variable had no START_DATE values, but had dates in the STOP_DATE column.

When discussing this with @vaskalan and s380 it turns out that:
> When entering a dataset with temporalitytype FIXED, we always add the date when the measurement was done to STOP_DATE for each row. The reason: The presentation of the variable in time becomes correctly displayed in ROSE.

In other words; this is an important convention to display the variable correctly in ROSE. So I added an extra check for this if temporalityType is FIXED.

Bumped version to 0.3.0. The plan is to bump the version to 1.0.0, once its implemented and tested in the dataset-builder pipeline.

### CODE REVIEW

File: microdata_validator/dataset_validator.py [hunk 1]
```diff
@@ -134,6 +134,11 @@ def __is_data_row_consistent(temporality_type: str, data_row: tuple,
                     f"DataSet.temporalityType is {temporality_type}"
                 )
             )
+        if stop is None or str(stop).strip(" ") == "":
```
Comment (pawbu) [inline microdata_validator/dataset_validator.py:137]: Isn't `... or str(stop).strip() == ""` enough?
https://docs.python.org/3.8/library/stdtypes.html#str.strip

Comment (DanielElisenberg) [reply microdata_validator/dataset_validator.py]: True! Will remove! Should we add validation that requires an empty start value as well 🤔 ? (Similar to the problem we saw in 1.5)

Comment (pawbu) [reply microdata_validator/dataset_validator.py]: I'm not sure about these rules, but if you think it's a similar case to 1.5, then we probably should have it here as well.

**Comment by DanielElisenberg** [comment]
### UPDATE
Added the rule to FIXED datasets present in 1.5:
* If temporality type is fixed, temporalCoverageStart is "1900-01-01"
Added validation error as well:
* If temporalitytype is fixed, return error if startdate is present in data

The validation rule is not present in 1.5, but its useful to be strict with this rule for a myriad of reasons (space efficiency, cleanliness of data, validation consistent with convention etc.)

File: microdata_validator/dataset_validator.py [hunk 2]
```diff
@@ -134,6 +134,17 @@ def __is_data_row_consistent(temporality_type: str, data_row: tuple,
...
+        if stop is None or str(stop).strip() == "":
+            return (
+                row_number,
+                f"Expected STOP-date when temporalityType is {temporality_type}"
+            )
+        if not (start is None or str(start).strip() == ""):
+            print(start)
+            return (
+                row_number,
+                f"There should no START-date when temporalityType is {temporality_type}"
```
Comment (pawbu) [inline microdata_validator/dataset_validator.py:146]: _There should **be** no ..._

**Review by pawbu — APPROVED** [review]
One comment, otherwise looks good

**Pull request merged**